tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_example_valid_invalid.py

```diff
@@ -35,55 +35,6 @@
             None,
             id="simple_eof_1_deploy",
         ),
-        pytest.param(
-            # Check that EOF1 undersize data is ok (4 declared, 2 provided)
-            # https://github.com/ipsilon/eof/blob/main/spec/eof.md#data-section-lifecycle
-            Container(
-                name="EOF1V0016",
-                sections=[
-                    Section.Code(
-                        code=Op.ADDRESS + Op.POP + Op.STOP,
-                        max_stack_height=1,
-                    ),
-                    Section.Data("0x0bad", custom_size=4),
-                ],
-            ),
-            "ef0001010004020001000304000400008000013050000bad",
-            EOFException.TOPLEVEL_CONTAINER_TRUNCATED,
```

Comment:
this exception is invalid, it actually returns no exception

Reply:
Did you try the latest evmone version? I'm seeing that it correctly returns this exception in the latest commit.